ci-operator/step-registry/ipi/install/libvirt/install/ipi-install-libvirt-install-commands.sh

```diff
@@ -126,16 +126,15 @@ do
   if [ "${BRANCH}" == "4.9" ] || [ "${BRANCH}" == "4.10" ] && [ "${ARCH}" == "ppc64le" ]; then
```

Comment:
I would update the condition here and remove `&& [ "${ARCH}" == "ppc64le" ]` instead of adding a separate block

Reply:
addressed in the recent commit.